main.js

```diff
@@ -1,5 +1,5 @@
-const readWorld = require('./readfiles.js');
+const readWords = require('./readfiles.js');
```

Comment:
It would be better to nave this function `readFiles` or something like it since it just reads files but not split them to words.